setup.py

```diff
@@ -83,6 +83,10 @@ def read(fname):
     packages = ["easybuild", "easybuild.easyblocks", "easybuild.easyblocks.generic"],
     package_dir = {"easybuild.easyblocks": "easybuild/easyblocks"},
     package_data = {'easybuild.easyblocks': ["[a-z0-9]/*.py"]},
+    scripts=[
+        # updated config.guess that is leveraged by the configuremake easyblock
+        'easybuild/scripts/config.guess',
```

Comment:
Maybe we should rename this script, or put it in a subdirectory, to clarify what it is intended for?